DependencyInjection/HttplugExtension.php

```diff
@@ -54,6 +66,10 @@ protected function configureClients(ContainerBuilder $container, array $config)
                 $first = $name;
             }
 
+            if (isset($config['_inject_collector_plugin'])) {
+                array_unshift($arguments['plugins'], 'httplug.collector.history_plugin');
```

Comment:
created #26 for this idea. i guess what we have now is ok, but the other thing sure would be cool.